static/sass/config/_layout.scss

```diff
@@ -0,0 +1,184 @@
+///       }
+///     }
+///   }
+@mixin grid {
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: $grid-columns-small;
+
+    @include above('medium-page') {
+      grid-template-columns: $grid-columns-large;
+    }
+  }
+}
+
+
+// Margin/Size Extension Variables will move to scale map
+// when Accountrement Scale is updated
+$full-content: 40rem;
+$extend-small: 70vw;
+$extend-medium: 80vw;
+$extend-large: 90vw;
+$extend-full: 100vw;
+
+
+// Extend thing outside its container on both sides
+// ================================================
+// viewport, minus the main content width, divided by 2, then make negative
+/// @group layout
+/// @link http://codepen.io/mirisuzanne/pen/ded4ae1f55f9b24b2aa0ea4aaac71e47?editors=0100
+///    Miriam's Sample CodePen
+/// @link http://www.sassmeister.com/gist/705b458628ee9ac28b415166e1a09f7c - 
+///    Sample Sassmeister SCSS/CSS with Accoutrement
+/// @link http://www.sassmeister.com/gist/8374287846277fdf3361537dfaac27a8 - 
+///    Sample Sassmeister SCSS/CSS without Accoutrement
+/// @parameter {Size} $extend-base-size - For consistency, options include
+///    $extend-small, $extend-large or $extend-full
+/// @example scss -
+///   .foo {
+///     @include extend($extend-small);
+///   }
+///   .bar {
+///     @include extend($extend-medium);
+///   }
+///   .baz {
+///     @include extend($extend-large);
+///   }
+@mixin extend(
+  $extend-base-size
+) {
+
+  $extend-large-size: $extend-base-size - 10vw;
+
+  @include above('medium-page') {
+    margin-left: calc((#{$extend-base-size} - #{$full-content}) / 2 * -1);
+    margin-right: calc((#{$extend-base-size} - #{$full-content}) / 2 * -1);
+    max-width: $extend-base-size;
+    width: $extend-base-size;
+  }
+
+  @if ($extend-base-size != $extend-full) {
+    @include above('large-page') {
+      margin-left: calc((#{$extend-large-size} - #{$full-content}) / 2 * -1);
+      margin-right: calc((#{$extend-large-size} - #{$full-content}) / 2 * -1);
+      max-width: $extend-large-size;
+      width: $extend-large-size;
+    }
+  }
+}
+
+
+// Extend thing outside its container on one side only
+// ===================================================
+// viewport, minus the main content width, divided by 2, then make negative
+/// @group layout
+/// @link http://codepen.io/mirisuzanne/pen/ded4ae1f55f9b24b2aa0ea4aaac71e47?editors=0100
+///    Miriam's Sample CodePen
+/// @link http://www.sassmeister.com/gist/705b458628ee9ac28b415166e1a09f7c - 
+///    Sample Sassmeister SCSS/CSS with Accoutrement
+/// @link http://www.sassmeister.com/gist/8374287846277fdf3361537dfaac27a8 - 
+///    Sample Sassmeister SCSS/CSS without Accoutrement
+/// @parameter {Side | String} $side - Choose right or left
+/// @example scss -
+///   .foo {
+///     @include extend-side(left);
+///   }
+///   .bar {
+///     @include extend-side(right);
+///   }
+@mixin extend-side(
+  $side
+) {
+
+  margin-bottom: size('pad');
```

Comment:
Since I am eliminating the need to use an additional utility class (.align-left, .align-right) I have added this bottom margin in order for the text that wraps the image mid-paragraph to not get too close/touch the image. I know @mirisuzanne commented above this margin-bottom in Slack yesterday.